docs/configuration.asciidoc

```diff
@@ -33,7 +33,10 @@ using Elastic.Apm.AspNetCore;
 
 public class Startup
 {
-    public Startup(IConfiguration configuration)
+
+    private readonly IConfiguration Configuration;
```

Comment:
we should rename this from `Configuration` to `_configuration`, in line with private field naming conventions. Line 41 would also need to change to `_configuration = configuration;`

Reply:
agree. I made a new commit.